src/Leaflet.VectorGrid.js

```diff
@@ -77,16 +77,16 @@ L.VectorGrid = L.GridLayer.extend({
 				for (var layerName in vectorTile.layers) {
 					this._dataLayerNames[layerName] = true;
 					var layer = vectorTile.layers[layerName];
-	
```

Comment:
Would be nice if we could back out all these changes which I guess are caused by some auto-setting in you editor - while I think all agree that we should not have trailing whitespace, the change history will be confusing if these changes are done as part of other commits.

Reply:
I backed out the whitespace changes